Clojure-Functions.md

```diff
@@ -0,0 +1,82 @@
+# Clojure - More on Functions
+Functions! They're pretty important. It's pretty difficult to do anything without
+a function. They're integral to any language, but especially Clojure, since it's
+a functional programming language that rejects object-oriented design. Let's
+learn some more about them!
+
+## Arity
+'Arity' refers to the number of arguments that your function expects.
```

Comment:
I updated my comment, but yeah I figured you were talkign about the term, but with single quotes I was not sure if you wanted to use the code block or just quote the word.